### DISCUSSION TITLE
Controlling remote alarms added in hik connect

### DISCUSSION THREAD

**Discussion by vishnup3211** [petrleocompel/hikaxpro_hacs#79] [Q&A] [answered]
hi, is there any way that we can control a hikvision ax hybrid alarm in a remote location through this.??

**Comment by petrleocompel** [comment]
Yes you can. But this is more of networking question.

**Reply by vishnup3211** [reply]
i mean if the only way to access the device is through hik-connect account ( no local access to the ip address of the device, using a 4g sim card for data access)

**Reply by petrleocompel** [reply]
No. You have to use some cloud based integration.

**Reply by trempetic92** [reply]
Added information in my newest Discussion topic.

**Reply by petrleocompel** [reply, marked as answer]
I am going to stay on my point. This integration is strictly "Local".
We are not relying on any "cloud" service and I am sadly for you guys not going to implement that.
I am unhappy about "cloud" integrations for reason.

You can still achieve "remote control" via having multiple HA instances on the sites. Or going crazy with networking to site-to-site VPNs tunnels and linking all to one HA. 

If you want "cloud" integration feel free to create "cloud-only" integration. Or there might be someones else integration.
But this repository is going to stay as "local" integration.
